mkdocs.yml

```diff
@@ -6,7 +6,7 @@ site_author: Chutney Testing Team
 repo_name:
 repo_url: https://github.com/chutney-testing/chutney/
 edit_uri: https://github.com/chutney-testing/chutney-testing.github.io/docs
-# copyright: © Chutney Testing 2017 | under the Apache License, Version 2.0
+copyright: © Chutney Testing 2017 | under the Apache License, Version 2.0
```

Comment:
Beware, you may have add files you didn't want to add and commit.

Remove changes to this file and `rebase` your branch from the main.